theseus/extlib/cusolver_lu_solver.cpp

```diff
@@ -88,21 +88,21 @@ CusolverLUSolver::CusolverLUSolver(int batchSize,
     const int *pA_rowPtr_cpu = A_rowPtr_cpu.data_ptr<int>();
     const int *pA_colInd_cpu = A_colInd_cpu.data_ptr<int>();
 
-    // we compute the permutation Q which allows 
+    // we compute the permutation Q which allows
```

Comment:
Seems like all changes to this file are just adding extra space to a few lines. If this is not automatic from a linter then we should remove this file from the PR.

Reply:
actually I was cleaning extraspaces, and ended up doing so with a few extra files.